### PR TITLE
#6598 display javadoc on shift-tab

### PR DESCRIPTION
**Pull request by rbidas**

### CODE REVIEW

File: doclet/build.gradle [hunk 1]
```diff
@@ -0,0 +1,25 @@
+group 'com.twosigma.beakerx'
```
Comment (scottdraves) [inline doclet/build.gradle]: banner

**Comment by scottdraves** [comment]
move doclet directory into kernel directory so it can share the same gradlew

**Comment by scottdraves** [comment]
i am hitting shift-tab with a classname open, but no window is open...
what am i missing?

**Comment by benmccann** [comment]
I was wondering if this is documented anywhere yet? If not, it would be good to add it to the docs as well

**Comment by scottdraves** [comment]
Cool, the help page opened up, but I only see the class name, not the actual doc?
Looks like it needs a bit more margin on top because the buttons are blocking it.

screen shot 2018-02-27 at 11 31 19 am 2

File: doclet/gradlew [hunk 2]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (scottdraves) [inline doclet/gradlew]: move the doclet into the kernel directory so that it shares the gradle stuff.

**Comment by scottdraves** [comment]
cool thx.  during compilation i am seeing a bunch of errors like this:
```
    Constructing Javadoc information...
    /Users/spot/src/beakerx/kernel/base/src/main/java/com/twosigma/beakerx/handler/KernelHandler.java:23: error: package com.google.common.base does not exist
    import static com.google.common.base.Preconditions.checkNotNull;
                                        ^
    /Users/spot/src/beakerx/kernel/base/src/main/java/com/twosigma/beakerx/handler/KernelHandler.java:23: error: static import only from classes and interfaces
```
we should not even being generating javadoc for internal classes, just for the user-visible runtime classes.  the regular javadoc had a filter for this...  or maybe the errors can just be fixed, idk.

**Pull request merged**